src/libs/SubscriptionUtils.ts

```diff
@@ -363,7 +363,11 @@ function calculateRemainingFreeTrialDays(): number {
     }
 
     const currentDate = new Date();
-    const diffInSeconds = differenceInSeconds(parseDate(lastDayFreeTrial, CONST.DATE.FNS_DATE_TIME_FORMAT_STRING, currentDate), currentDate);
+    const lastDayDate = new Date(lastDayFreeTrial);
+    const lastDayFreeTrialDateUTC = new Date(
```

Comment:
The changes work but I think we need to change the variable name  `lastDayFreeTrialDateUTC` as the time represents the user’s timezone specific time. And we compare this with the user’s current time based on his timezone. 

Alternatively, if we want to use UTC time comparison, we can convert current date to UTC time as follows and replace all occurrences of `currentDate` usage. This may be neater too.

    const currentDateinUTC = new Date(currentDate.getUTCFullYear(), currentDate.getUTCMonth(), currentDate.getUTCDate(), currentDate.getUTCHours(), currentDate.getUTCMinutes(), currentDate.getUTCSeconds());